README.md

```diff
@@ -1,16 +1,185 @@
+
+Ultimately nested data structures allow us to model complex data in a way that
+humans understand **and** that computers can speedily process. In the rest of this
+lesson, we're going to talk about some of the specifics around how NDS' fulfill
+this goal.
+
+## Nested Data Structures Are Easier for Humans to Understand
+
+Can you describe the nested `Array` above in English?
+
+Try it yourself before you try ours.
+
+Tried it? OK, here's our attempt:
+
+> "The `Array` called `array_of_arrays` has two elements. One element is an
+> `Array` containing 3 elements. Those elements are the first odd numbers greater
+> than `0`. The other element in `array_of_arrays` contains 3 elements that are
+> the first even numbers greater than 0."
+
+Is that _fun_ to read? Not really. Is it easy to think about? Not really.
+
+Now, imagine what the
+paragraph to describe agricultural output of South America looks like.  It makes
+us want to run away screaming.
+
+Humans
+learned long ago  that reading complex paragraphs like those is **not** a real strength of
+ours (Daniel Kahnemann in his book _Thinking Fast and Slow_ discusses this at
+length.).
+
+The human animal doesn't like to think methodically (like a computer). When we
+encounter complex paragraphs we skip sentences, we look ahead, etc.  We might
+even have to take notes, or track the words with a pencil to keep our focus.
+
+Humans have learned that we can share information like this far better in a bar
+chart, a line graph, or a pie chart. Economic outputs are perfect for _tables_.
+
+Unfortunately, computers don't understand pie charts. But if we structure our
+data in NDS', we can get something that's easier for humans to read **and**
+that the computer can work with.
+
+Provided you understand `Array` literal syntax, your brain loves seeing data in
+this visually "rhythmic" way.  The NDS uses space and patterns to communicate _both_ the
+content inside, as well as describe some  rules (_In this `Array` there are
+`Array`s. It seems like most inner `Array`s have three elements._).
+
+For clarity in thinking and communicating, working with NDS' is a vital skill.
+
+## Nested Data Structures Are Usable by Computers
+
+We said it before, but a computer can't work with a pie chart or a table. The
+closest approximation it can work with is a nested data structure. So NDS' make
+it _easier_ for humans to understand complex data, but they're the _only_ way a
+computer can understand complex data.
+
+## Nested Data Structures Are Transformed to Produce Insight
+
+Early (1940s-1980s) programmers pretty much had one job description:
+
+> "Take a bunch of raw data and output a re-organized version of it, or do a
+> calculation from all of it."
+
+In fact, the complex world of programming was often housed in a department
+called "data processing."
+
+Imagine you're back in that early era of "data processing." Let's now suppose
+someone asks us to:
+
+> Get the element in the `1` index for all the nested data.
```

Comment:
It feels like the array of arrays needs to be before this. Maybe just having the AoA code snippet at the beginning of this section would help.